File: lol.go
package main

import "fmt"

func main() {
    fmt.Println("Hello, Go!")
    testFunc()
}

// Unused function (golangci-lint will flag this)
func unusedFunction() {
    fmt.Println("This function is not used")
}

// Function with linting issue (missing comment)
func testFunc() {
    fmt.Println("Testing Go linting")
}

File: lol.js
function hello() {
    console.log("Hello, JavaScript!")
}

hello();

// Unused variable (ESLint will flag this)
let unusedVar = 42;

// Indentation issue (ESLint will flag this)
if(true){
console.log("Indentation issue detected!");
}

// Missing semicolon (ESLint will flag this)
console.log("Linting Test")
